ubi8-py36/Dockerfile

```diff
@@ -26,7 +26,8 @@ COPY ./s2i_assemble.patch /tmp/s2i_assemble.patch
 RUN TMPFILE=$(mktemp) && \
     TMPFILE_ASSEMBLE=$(mktemp) && \
     pushd "${STI_SCRIPTS_PATH}" && patch -p 1 </tmp/s2i_assemble.patch && popd && \
-    /usr/bin/pip3 install micropipenv[toml]==1.0.1 thamos==1.3.1 && \
+    /usr/bin/pip3 install -U "pip==20.1.1" && \
```

Comment:
This will update system pip, not the one that is used during the s2i build process. Hence installing packages will fail as before.